src/message/basicMessage.js

```diff
@@ -143,6 +144,24 @@ class BasicMessage{
         this.apiTemplate = value;
     }
 
+     /**
+     * Sets the ampBody for the message.
+     * (Optional) Either textBody or HtmlBody must be used or use a ApiTemplate with an AmpBody
```

Comment:
rephrase this line: "Either TextBody or HtmlBody must be used with the AmpBody or use a ApiTemplate"